portal-api/src/v1/controllers/facilities.controller.js

```diff
@@ -128,33 +116,30 @@ const queryAllFacilities = async (
       {
         $sort: {
           ...sort,
-          updatedAt: -1,
+          updatedAt: -1, // TODO SR-8: Sort order -- no $eq expression needed
         },
       },
       {
         $facet: {
           count: [{ $count: 'total' }],
           facilities: [
-            { $skip: startPage },
-            ...(pagesize ? [{ $limit: pagesize }] : []),
+            { $skip: startPage }, // TODO SR-8: Does not take an expression -- no $eq expression needed
+            ...(pagesize ? [{ $limit: pagesize }] : []), // TODO SR-8: Does not take an expression -- no $eq expression needed
```

Comment:
Remove todo once noted:
```suggestion
            { $skip: startPage },
            ...(pagesize ? [{ $limit: pagesize }] : []),
```